src/gettingstarted/first-project.md

```diff
@@ -0,0 +1,91 @@
+# Your first project
```

Comment:
IMO this is too verbose, without enough breaks or reference points.

Perhaps there is some unnecessary detail. Regardless, I think adding hyperlinks from each concept within the prose (to their relevant detailed docs) would break it up for the eye.

Reply:
Fixed by converting it to a shorter set of bullet points.